service-python/pdfgenerator/src/lib/queues.py

```diff
@@ -31,8 +31,7 @@ def on_generate_callback(channel, method, properties, body):
         # logging.info(f"Variables: {variables}")
         template = pdf_generator.generate_template_file(diagnosis_name, variables)
         pdf = pdf_generator.generate_pdf_from_string(template)
-        redis_client.save_data(claim_id, base64.b64encode(pdf).decode("ascii"))
-        redis_client.save_data(f"{claim_id}_type", diagnosis_name)
+        redis_client.save_hash_data(f"claim-{claim_id}", mapping={"pdf": base64.b64encode(pdf).decode("ascii"), "type": diagnosis_name})
```

Comment:
Since the hash contents is only about the pdfgenerator, how about these keys:
```suggestion
        redis_client.save_hash_data(f"{claim_id}-pdf", mapping={"contents": base64.b64encode(pdf).decode("ascii"), "diagnosis": diagnosis_name})
```